lib/app/cache.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:dio/dio.dart';
+import 'package:dio_http_cache/dio_http_cache.dart';
+
+class CacheManager {
+  static Options todayCards({bool refresh = true}) =>
+      buildCacheOptions(Duration(seconds: 15),
+          maxStale: Duration(days: 7), forceRefresh: refresh);
```

Comment:
```suggestion
  static Options todayCards({bool refresh = true}) => buildCacheOptions(
        Duration(seconds: 15),
        maxStale: Duration(days: 7),
        forceRefresh: refresh,
      );
```

Formatting suggestion